utils/users.js

```diff
@@ -43,6 +43,46 @@ const getUsername = async (userId) => {
     throw error;
   }
 };
+
+/**
+ * Used for receiving username when providing userId, if not found then returns undefined
+ *
+ * @param userId {String} - userId of the User.
+ * @returns username {String} - username of the same user
+ */
+const getUsernameElseUndefined = async (userId) => {
+  try {
+    const {
+      user: { username },
+    } = await fetchUser({ userId });
+    return username;
+  } catch (error) {
+    logger.error("Something went wrong", error);
+    return undefined;
+  }
+};
+
+/**
+ * Used for receiving userId when providing username, if not found then returns undefined
+ *
+ * @param username {String} - username of the User.
+ * @returns id {String} - userId of the same user
+ */
+
+const getUserIdElseUndefined = async (username) => {
```

Comment:
better name could be getUserId

Reply:
explained for the same in getUsernameElseUndefined.